sentry-android-core/proguard-rules.pro

```diff
@@ -12,9 +12,10 @@
 #-keep class com.google.gson.stream.** { *; }
 
 # Application classes that will be serialized/deserialized over Gson
--keep class io.sentry.core.** { <fields>; }
--keep class io.sentry.core.protocol.** { <fields>; }
+-keep class io.sentry.core.** { *; }
```

Comment:
@bruno-garcia I'd like to remove this exception, so lib may be optimized, but for that, we need to move some data classes from io.sentry.core to io.sentry.core.protocol eg Breadcrumb

Reply:
Can we just add the exact classes instead of moving them?
I mean since you need to import each individual class in Java, we can move them, doesn't matter.
Some types stayed in the top level namespace in .NET to to available without adding a bunch of `import`

Reply:
jep, also all the public classes should be there.
lets merge it for now